instagram/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,126 @@
+import React, { Component } from 'react';
+// import PropTypes from 'prop-types';
+import PostContainerList from '../PostContainer/PostContainerList';
+import Header from '../Header/Header';
+import './Style.css';
+
+class PostsPage extends Component {
+  constructor(props){
+    super(props);
+    this.dummyData = props.dummyData;
+    this.state = {
+      usr: props.loginUsr,
+      data: [],
+      newComment: '',
+      searchText: ''
+    };
+  }
+
+  componentDidMount() {
+    this.setState({data: this.dummyData});
+  }
+
+  handleChange = e => {
```

Comment:
This was eye opening for me, good job on figuring the binding issue we were working on.